modules/web/server/src/main/scala/observe/web/server/logging/AppenderForClients.scala

```diff
@@ -20,7 +20,8 @@ import observe.model.events._
  *
  * This is out of the scala/http4s loop
  */
-class AppenderForClients(out: Topic[IO, ObserveEvent]) extends AppenderBase[ILoggingEvent] {
+class AppenderForClients(out: Topic[IO, ObserveEvent])(implicit ioruntime: IORuntime)
```

Comment:
Should we use `Dispatcher` here instead?

Reply:
Yes, we should. I'll change it.